src/documents/Priorities.md

```diff
@@ -0,0 +1,107 @@
+
+```javascript
+fluid.defaults("fluid.uploader.errorPanel.section", {
+    gradeNames: ["fluid.viewComponent"],
+...
+    listeners: {
+        "onCreate.bindHandlers": {
+            funcName: "fluid.uploader.errorPanel.section.bindHandlers",
+            priority: "before:refreshView"
+        },
+        "onCreate.refreshView": "{that}.refreshView"
+    }
+});
+```
+
+Amongst the two `onCreate` listeners, the framework guarantees that the `bindHandlers` listener will always be invoked before `refreshView`.
+
+## Table of supported sites for priorities
+
+The following table lists all the locations in component options where priorities are supported, together with some notes about their limitations and interpretation.
+
+<table>
+  <thead>
+    <tr>
+      <td>Options Record Entry</td>
+      <td>Polarity</td>
+      <td>Notes</td>
+    </tr>
+  </thead>
+  <tr>
+  <td><a href="InfusionEventSystem.md#registering-a-listener-to-an-event"><code>listeners</code></a></td>
+  <td>Lower numeric numbers, <code>last</code>, and <code>after</code> will fire later than others</td>
+  <td>Full support for constraints, uniquifying based on <code>namespace</code></td>
+  </tr>
+  
+  <tr><td><a href="ChangeApplierAPI.md#model-listener-declaration"><code>modelListeners</code></a></td>
+  <td>Ordering as for <code>listeners</code>. Model listeners will be sorted globally across the entire <a href="ModelRelay.md">model skeleton</a> by priority when a transaction concludes.</td>
+  <td>Full support for constraints, no uniquifying based on <code>namespace</code> (currently)</td></tr>
+  
+  <tr><td><a href="IoCSS.md"><code>distributeOptions</code></a></td>
+  <td>Lower numeric numbers, <code>last</code> and <code>after</code> will appear <strong>later</strong> in the merge order (with merging considering to occur from left to right &#8212; note that this is the opposite direction in which
+  parent <code>gradeNames</code> of a grade are considered), that is, these will represent <strong>stronger</strong> options that will merge on top of other options.</td>
+  <td>Full support for constraints, no uniquifying based on <code>namespace</code> (currently). Note that priority for distributions with the priority field left blank will be determined by component tree 
+  topology &#8212; distributions which travel a further distance will have weaker ("earlier", "lower") priority than distributions from nearby. Numeric priorities should not be used.</td>
+  </tr>
+  <tr><td><a href="SubcomponentDeclaration.md"><code>components</code></a></td>
+  <td>Ordering as for <code>listeners</code>. Components with lower priority will be constructed later.</td>
+  <td>Note that there is <strong>no</strong> support for <code>namespace</code> for component declarations. The use of priorities for subcomponent declarations is <strong>not recommended</strong>. You should always be able 
```

Comment:
No support for namespaces implies no support for constraints, yes - since constraints use the namespace names for references. I've adjusted this section a little to account for the fact that it describes both support for namespaces in themselves as well as the priorities that might be derived from them.